test/densities/box_setup/particles_pi_k_n_lambda.txt

```diff
@@ -0,0 +1,32 @@
+# NAME MASS[GEV] WIDTH[GEV] PARITY PDG
+
+########## unflavored mesons ##########
+
+π               0.138   7.7e-9    -      111     211
+η               0.548   1.31e-6   -      221
+σ               0.800   0.400     +  9000221
+ρ               0.776   0.149     -      113     213     # mass required by EOS code
+
+########## strange mesons ##############
+
+K         0.494  0      -     311    321
+K*(892)   0.892  0.050  -     313    323
+
+########## N baryons ###################
+
+N       0.938 0     +      2112     2212
+N(1440) 1.440 0.350 +     12112    12212
+N(1520) 1.515 0.110 -      1214     2124
+N(1535) 1.530 0.150 -     22112    22212
+N(1650) 1.650 0.125 -     32112    32212
+
+########## Δ baryons ###################
+
+Δ       1.232 0.117 +   1114    2114    2214    2224
+
+########## Λ baryons ###################
+
+Λ        1.116 0      +     3122
+
+########## Σ baryons ###################
+Σ       1.189 0     +    3112    3212    3222
```

Comment:
Why are there Delta and Sigma baryons in the `pi_K_N_lambda` box? Also why is there a Delta decay in the Decaymodes list? Or should the target rather be a `pi_K_N_lambda_Delta_Sigma` target?

Reply:
Somehow in combination with the decaymodes these particles are needed so I added them... For this purpose it shouldn't make a difference. Maybe ```pi_K_N+Res``` is more suitable? ```pi_K_N_lambda_Delta_Sigma``` is a bit unhandy ;)

Reply:
I added a corresponding comment in the particle lists.txt. Those resonances are needed for decays of the others, that's why they are in there.